docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1268,180 +1268,193 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
+
+* When the name of the container does not match the application name (or the overridden name), container specific configuration will be ignored.
 ====
 
 == Service Binding [[service_binding]]
 
-Quarkus supports binding services to applications via the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
-Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing
-applications running in appropriately configured Kubernetes clusters to consume services (such as a Database or a Broker) without the need for user configuration.
+Generally, OpenShift applications and services, also referred to as deployable workloads, need to be connected to other services for retrieving additional information such as service URLs or credentials.
+
+The required communication for obtaining this information is managed by the Service Binding Operator, which also determines:
 
-Currently, the following Quarkus extensions (and extensions that depend on these) support this feature:
+* how a service consumer is intended to bind to such a service
+* tools for application and service binding, such as Service Binding Operator
 
-* quarkus-jdbc-mariadb
-* quarkus-jdbc-mssql
-* quarkus-jdbc-mysql
-* quarkus-jdbc-postgresql
-* quarkus-kafka-client
-* quarkus-smallrye-reactive-messaging-kafka
-* quarkus-mongo-client
 
-This list of extensions will grow as more services with supported bindings become available on Kubernetes.
+Quarkus supports binding services to applications using the link:https://github.com/k8s-service-bindings/spec[Service Binding Specification for Kubernetes].
+Specifically Quarkus implements the link:https://github.com/k8s-service-bindings/spec#workload-projection[Workload Projection] part of the specification, thus allowing applications running in appropriately configured Kubernetes clusters to consume service, such as a Database or a Broker, without the need for user configuration.
 
 To enable Service Binding support, in addition to one of the currently supported extensions, the `quarkus-kubernetes-service-binding` extension needs to be added to the application dependencies.
 
-=== Example using PostgreSQL
+=== Workload projection
 
-If the Kubernetes cluster in which your Quarkus application runs supports provisioning a PostgreSQL database, one could use the following dependencies to take advantage of the Service Binding feature:
+Workload Projection is the process of projecting binding information to a specific workload, and a way of externalizing configuration. The projection is usually applied as a directory structure that follows certain conventions and can be added to an application or a service using volume mounts. Afterwards, thanks to the `kubernetes-service-binding` extension, the projective binding can be used as a configuration source, allowing the configuration of additional modules such as:
```

Comment:
```suggestion
Workload Projection is the process of obtaining configuration for services from the Kubernetes cluster. The projection is usually applied as a directory structure that follows certain conventions and can be added to an application or a service using volume mounts. This directory structure is used by the `kubernetes-service-binding` extension, to create configuration sources, allowing the configuration of additional modules such as:
```

Reply:
Will add in the upcoming batch :) Thank you.